src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -10316,7 +10316,7 @@ public override void M1<T>(System.Nullable<T> x)
             Assert.True(m1.OverriddenMethod.Parameters[0].Type.IsNullableType());
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/dotnet/csharplang/issues/2378#issuecomment-479634969")]
```

Comment:
>[Fact(Skip = "https://github.com/dotnet/csharplang/issues/2378#issuecomment-479634969")] [](start = 8, length = 88)

I think skipping tests doesn't help us to understand what would be the behavior after this PR. I would suggest to unskip the tests that existed prior to this PR and undo any modifications to the scenarios (like addition of constraints etc.). It is OK to add new skipped tests, but they should be linked to an active compiler issue (not a csharplang issue).